src/scss/_global.scss

```diff
@@ -34,7 +36,14 @@ $paragraph-line-height-tablet: 1.5rem;
 $paragraph-line-height-desktop: 1.5rem;
 
 // Structure Variables
-$content-margin: 1rem;
+$content-margin: 2rem;
+
+// Table Styles
+
+tbody {
+  font-family: monospace;
```

Comment:
I've tested several monospace fonts and Roboto Mono seems good. Dan also agreed it was a nice generic, readable monospace option.

https://www.google.com/fonts/specimen/Roboto+Mono